src/DBmysql.php

```diff
@@ -1310,18 +1310,24 @@ public static function quoteValue($value)
      */
     public function buildInsert($table, $params)
     {
-        $query = "INSERT INTO " . self::quoteName($table) . " (";
+        $query = "INSERT INTO " . self::quoteName($table) . ' ';
 
         $fields = [];
-        foreach ($params as $key => &$value) {
-            $fields[] = $this->quoteName($key);
-            $value = $this->quoteValue($value);
-        }
+        if (array_key_exists(0, $params) && count($params) === 1 && $params[0] instanceof QuerySubQuery) {
```

Comment:
Maybe it could be simplier to accept `array[QuerySubQuery`.

```suggestion
        if ($params instanceof QuerySubQuery) {
```